src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,12 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	public String getFeedbackToUser() {
+		return feedbackToUser;
+	}
+
+	public void setFeedbackToUser(String feedbackToUser) {
```

Comment:
As discussed in tutorial, the feedbackToUser seems like a immutable value (i.e. never change once we set it), so setter may not be appropriate in this case.